pkg/sql/logictest/testdata/logic_test/alter_table

```diff
@@ -2306,8 +2356,8 @@ FROM (
 LEFT JOIN pg_catalog.pg_depend r ON l.table_id = r.objid;
 ----
 table_id  name               state   refobjid
-205       test_serial_b_seq  PUBLIC  204
-204       test_serial        PUBLIC  NULL
+207       test_serial_b_seq  PUBLIC  206
+206       test_serial        PUBLIC  NULL
```

Comment:
These and the others all get bumped because my subtest non-transactionally increments the descriptor ID counter twice.

Reply:
why don't we put your subtest at the end of the file?

Reply:
In a sane world I would, however, doing so often causes annoying merge conflicts when backporting.